include/mdw_dslash5_tensor_core.cuh

```diff
@@ -8,6 +8,9 @@
 #include <math_helper.cuh>
 #include <shared_memory_cache_helper.cuh>
 
+#define THRUST_IGNORE_CUB_VERSION_CHECK
```

Comment:
I think we should avoid direct inclusion of `cub/cub.cuh` in QUDA code, instead use `cub_helper.cuh`.